package/sap-installation-wizard.spec

```diff
@@ -60,13 +60,15 @@ Authors:
 Summary:        Installation wizard for SAP Business One applications
 License:        GPL-2.0+
 Group:          System/YaST
-Version:        5.0.1
+Version:        5.0.2
 Release:        0
 PreReq:         /bin/mkdir %fillup_prereq yast2
 BuildRequires:  yast2
 BuildRequires:  sapconf
 Requires:       autoyast2
 Requires:       autoyast2-installation
+Requires:       patterns-sap-hana
```

Comment:
to be honest I do not get logic. For SP6 this dependency was removed and here it was added.